python_modules/dagster/dagster_tests/core_tests/graph_tests/test_graph.py

```diff
@@ -991,3 +994,17 @@ def my_graph_with_nesting(x):
     result = my_graph_with_nesting.execute_in_process(run_config={"inputs": {"x": {"value": 2}}})
     assert result.success
     assert result.output_for_node("my_graph.my_op") == 4
+
+
+def test_nothing_inputs_graph():
+    @op(ins={"sync_signal": In(Nothing)})
+    def my_op():
+        ...
+
+    @graph(input_defs=[InputDefinition("sync_signal", Nothing)])
+    def my_pipeline(sync_signal):
+        my_op(sync_signal)
+
+    the_job = my_pipeline.to_job()
+    result = the_job.execute_in_process()
+    assert result.success
```

Comment:
test passing `None` is as well

Reply:
we don't have a way to directly specify input values unfortunately

Reply:
^ actually this doesn't make sense so just ignore